src/Rebing/GraphQL/routes.php

```diff
@@ -78,37 +88,53 @@
         }
         else
         {
-            $router->get($queryRoute, ['uses'  => $queryController]);
-            $router->post($queryRoute, ['uses' => $queryController]);
+            $router->get($queryRoute, [
+                'uses' => $queryController,
+                'as'   => 'graphql.query',
+            ]);
+            $router->post($queryRoute, [
+                'uses' => $queryController,
+                'as'   => 'graphql.query.post',
+            ]);
         }
     }
 
-    // Mutation
-    if($mutationRoute)
+    // Mutation routes (define only if different than query)
+    if($mutationRoute && $mutationRoute !== $queryRoute)
```

Comment:
This change is unrelated to the PR but using named routes now made it obvious (well, at least to me) that there's no need to define the mutation routes explicitly if they're in fact the same as the query routes.

Otherwise the route names override each other we will _always_ end up with `graphql.mutation.*` route names, no matter what 🤷‍♀️